glue/sales_val_flagging.py

```diff
@@ -523,9 +523,9 @@ def get_parameter_df(
         "ptax_sd": [ptax_sd],
         "rolling_window": [rolling_window],
         "time_frame": [time_frame],
-        "short_term_owner_threshold": [short_term_thresh],
-        "min_group_thresh": [min_group_thresh],
-        "raw_price_thresh": [raw_price_thresh],
+        "short_term_owner_threshold": [short_term_threshold],
+        "min_group_thresh": [min_group_threshold],
```

Comment:
This min_group_thresh should eventually be migrated to min_group_threshold to maintain naming style.